launch/win_camera.launch.py

```diff
@@ -21,18 +21,25 @@ def generate_launch_description():
             parameters=[
                 {'frame_rate': 5.0},
                 {'frame_id': 'camera'},
+                {'image_width': 1280 },
+                {'image_height': 720 },
+                {'videoDeviceId': '\\\\?\\USB#VID_045E&PID_097D&MI_00#7&16658bf3&0&0000#{e5323777-f976-4f5b-9b55-b94699c46e44}\\GLOBAL'},
```

Comment:
the camera symbolic link is not guaranteed to be same/identical all the time.

Reply:
Oops, I didn't want to check that in